Formula/semgrep.rb

```diff
@@ -147,7 +147,8 @@ def install
       # Manually run steps from `opam exec -- make setup` to link Homebrew's tree-sitter
       system "opam", "update", "-y"
 
-      # We pass --no-depexts so as to disable the check for pkg-config. It seems to not be found when building on ubuntu
+      # We pass --no-depexts so as to disable the check for pkg-config. 
```

Comment:
```suggestion
      # We pass --no-depexts so as to disable the check for pkg-config.
```
Oops. @BrewTestBot really doesn't like changes that come from the GitHub UI...